packages/p2p/src/components/modal-manager/modals/order-details-confirm-modal/__tests__/order-details-confirm-modal.spec.tsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import { fireEvent, render, screen, waitFor } from '@testing-library/react';
-import { useStores } from 'Stores';
-import OrderDetailsConfirmModal from '../order-details-confirm-modal.jsx';
+import { StoreProvider, mockStore } from '@deriv/stores';
 import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+import { useStores } from 'Stores';
```

Comment:
just a doubt, import should be from '@deriv/stores'?
here and in the component?

Reply:
no. its a different store. p2p specific.